tests/plotting/test_nx_graph_export.py

```diff
@@ -0,0 +1,195 @@
+from typing import Any, cast
+
+import pandas as pd
+import pytest
+from rdkit import Chem
+from rdkit.Chem.rdDistGeom import EmbedMolecule
+
+from prolif.plotting.network.lignetwork import LigNetwork
+
+
+@pytest.fixture
+def simple_ligand_mol() -> Chem.rdchem.Mol:
```

Comment:
nitpick: for typing just use `Chem.Mol`, no need for `Chem.rdchem.Mol` (even if under the hood that's what Chem.Mol points to). It appears a couple more times below